app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/SearchResultsAdapter.java

```diff
@@ -44,54 +44,21 @@ public SearchResultHolder onCreateViewHolder(ViewGroup parent, int viewType) {
 
     @Override
     public void onBindViewHolder(final SearchResultHolder holder, int position) {
-        Datum datum = datumList.get(position);
+        final Datum datum = datumList.get(position);
+        holder.previewImageView.setVisibility(View.GONE);
         if (datum != null) {
             holder.titleTextView.setText(Html.fromHtml(datum.getTitle()));
             holder.descriptionTextView.setText(Html.fromHtml(datum.getDescription()));
-            if (!TextUtils.isEmpty(datum.getLink())) {
-                LinkPreviewCallback linkPreviewCallback = new LinkPreviewCallback() {
```

Comment:
@amitiwary999 In the current implementation of rss in app, we are 'not' searching anything. We get a list of data (search results) in the susi response response, right? Also, there is a link in the data for each search result. We are just displaying a preview of that link. Why do you want to remove that preview?